lib/v8debugapi.js

```diff
@@ -485,18 +484,20 @@ module.exports.create = function(logger_, config_, fileStats_) {
         }
       }
     }
-    var localConfig;
     if (breakpoint.action === 'LOG') {
-      localConfig = { capture: { maxFrames: 0 } };
-      _.defaultsDeep(localConfig, config);
+      var frame = execState.frame(0);
+      var evaluatedExpressions = [];
+      breakpoint.expressions.forEach(function(exp) {
+        evaluatedExpressions.push(state.evaluate(exp, frame).mirror.value());
+      });
+      breakpoint.evaluatedExpressions = evaluatedExpressions;
```

Comment:
I think you will need to omit the `evaluatedExpressions` loop above, or perhaps, integrate this code in the loop above. At any rate, looking at the code, I think the expectation for log points would be that we do not re-order the list of evaluated expression. The call to splice above changes this?